pkg/combustion/kubernetes_test.go

```diff
@@ -186,6 +191,102 @@ func TestConfigureKubernetes_SuccessfulRKE2Server(t *testing.T) {
 	assert.Equal(t, "cilium", configContents["cni"], "default CNI is not set")
 }
 
+func TestConfigureKubernetes_SuccessfulRKE2ServerWithLocalManifests(t *testing.T) {
```

Comment:
I'd suggest to reuse the full successful example instead of repeating 85% of it for the manifests scenario.

Reply:
Yeah was think that as well, will update 👍